bin/main.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
+
+require_relative '../lib/player.rb'
+require_relative '../lib/board.rb'
+
+puts "player one's name ..."
+name = gets.chomp
+player_one = Player.new(name, 'X')
+
+puts
+puts "player two's name ..."
+name = gets.chomp
+player_two = Player.new(name, 'O')
+puts
+
+puts "#{player_one.name} is #{player_one.mark}"
+puts "#{player_two.name} is #{player_two.mark}"
+puts
+
+board = Board.new
+
+board.display_board
+
+def valid_number(move)
+  while move.nil? || move.to_i < 1 || move.to_i > 9
+    puts 'Please insert a valid number (1-9)'
+    move = gets.chomp.to_i
+  end
+  move
+end
+
+def check_invalid_slot(move, board)
+  while board.booked_slots.any?(move)
+    puts 'Please choose another slot'
+    move = gets.chomp.to_i
+    move = valid_number(move)
+  end
+  move
+end
+
+def switch_user(player_turn, board, move_counter)
+  puts "#{player_turn.name}'s move"
+  move = gets.chomp.to_i
+  move = valid_number(move)
+  move = check_invalid_slot(move, board)
+  board.booked_slots.push(move)
+  board.mark_on_board(move, player_turn.mark)
+  if move_counter >= 5 && board.win_check(player_turn.mark)
+    puts "#{player_turn.name} is winner"
+    return false
+  end
+
+  if move_counter >= 5 && move_counter == 9
```

Comment:
There's an issue with a draw, After a draw game, the game still running and requesting a move from the next player.
You can ask the user if they want to initialize a new game.
![Screen Shot 2020-07-20 at 6 32 42 PM](https://user-images.githubusercontent.com/52430951/87996391-a4be9200-cab7-11ea-86cd-ed6421da4e95.png)